package.json

```diff
@@ -79,7 +80,8 @@
     "ts-md5": "^1.2.8",
     "tslib": "^2.3.0",
     "uuid": "^8.3.2",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.11.4",
+    "pa11y-ci":" ^3.0.1"
```

Comment:
You should move this to the next section, `devDependencies`. Dependencies are intended to be delivered with the app, i.e., the bits would get delivered to dockstore.org. Dev dependencies are intended for development/building, and not delivered with the app.

Reply:
Corrected. Thanks for the explanation.